resources/blog/posts/deploy-laravel-application-to-amazon-ec2-instance.md

```
@@ -0,0 +1,225 @@
+<p> <img src="https://cdn.hashnode.com/res/hashnode/image/upload/v1691821858878/c66b8705-addd-4f4d-8262-b122293bd813.png" alt class="image--center mx-auto" /></p>
+</li>
+<li><p>Now you are on the Lunch instance page:</p>
+<p> <img src="https://cdn.hashnode.com/res/hashnode/image/upload/v1691822561677/f3f5d1c6-86cc-43b2-b200-1787d5779190.png" alt class="image--center mx-auto" /></p>
+<p> A. Give your instance a name<br /> B. Select Ubuntu<br /> C. Select "Ubuntu Server 22.04 LTS" AMI</p>
+</li>
+<li><p>In the next section:</p>
+<p> <img src="https://cdn.hashnode.com/res/hashnode/image/upload/v1691822921532/aaa75dbd-6169-4f63-a88d-7503c3fb1095.png" alt class="image--center mx-auto" /></p>
+<p> D. Select the instance type; I selected a free-tier eligible "t2.micro".<br /> E. Select the key pairs you selected previously.</p>
+</li>
+<li><p>Select all of these three checkboxes:</p>
+<p> <img src="https://cdn.hashnode.com/res/hashnode/image/upload/v1693050910149/cf9d94dd-b3d3-4351-9be2-0ed7eb0ea287.png" alt class="image--center mx-auto" /></p>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)